Services/Interfaces/web_interface/advanced_controllers/tentacles.py

```diff
@@ -18,14 +18,33 @@
 
 import tentacles.Services.Interfaces.web_interface.advanced_controllers as advanced_controllers
 import tentacles.Services.Interfaces.web_interface.login as login
+from tentacles.Services.Interfaces.octo_ui2.models.octo_ui2 import import_cross_origin_if_enabled, dev_mode_is_on
 import tentacles.Services.Interfaces.web_interface.util as util
 import tentacles.Services.Interfaces.web_interface.models as models
 import octobot_commons.authentication as authentication
 
-
-@advanced_controllers.advanced.route("/tentacles")
-@login.active_login_required
-def tentacles():
+tentacles_route = "/tentacles"
```

Comment:
Could you wrapp this code (the as tentacles-package route) in an annotation or a function to prévient duplicating code?